tests/conftest.py

```diff
@@ -4,8 +4,36 @@
 import sys
 from os.path import abspath, dirname, join
 
+import pytest
+
 
 # allow having multiple repository checkouts and not needing to remember to rerun
 # 'pip install -e .[dev]' when switching between checkouts and running tests.
 git_repo_path = abspath(join(dirname(dirname(__file__)), "src"))
 sys.path.insert(1, git_repo_path)
+
+# import local modules after fixing up sys.path
+if 1:  # flake be quiet
```

Comment:
Not sure if this is the best practice. I believe `flake` can be temporarily disabled by some magical comment?

Reply:
I looked at different options discussed on [SO](https://stackoverflow.com/questions/36827962/pep8-import-not-at-top-of-file-with-sys-path). This seemed to be the simplest, most intuitive and easily expandable in the future approach as new imports will be added there. You can add `# noqa` after such imports, but you will have to do it for each import, whereas this proposed in PR approach requires just one line of noise.

As far as best practice, style enforcers are awesome in a large collaboration project, until they don't.

I'm, of course, open to other solutions, that seem to be agreeable to most.

Reply:
why do you need need to hack sys.path? you cant import transformers.utils from here?

Reply:
because I have many repo clones and this ensures the tests are always run against the correct local-to-repo modules and not the env-wide installed modules. Does it make sense? Otherwise I always have to do `PYTHON_PATH="./src/" pytest` which is a waste of time, when it can be done automatically internally.

If we didn't have that `src` dir (why do we have it?) then it wouldn't have been needed, since "." is in the path.